src/components/editor/faultTree/Editor.tsx

```diff
@@ -28,7 +28,7 @@ const Editor = ({ setAppBarName }: DashboardTitleProps) => {
   const [showSnackbar] = useSnackbar();
   const [requestConfirmation] = useConfirmDialog();
 
-  const [faultTree, refreshTree, rootReqProb] = useCurrentFaultTree();
```

Comment:
Wasn't it already removed in #248 ?

Reply:
Yes, you're right, my bad. Can we leave this here to avoid conflicts?